### ISSUE TITLE
Update: Change home page to previous Parralax design

### ISSUE THREAD

**Issue by jonpmarino** [reynalddizon/MapStoryToDo#17]

**Comment by travislbrundage** [comment]
This one has been a bit hard for me to tell. I think it's about a day of effort - it looks partially implemented already, but with a library I am unfamiliar with. I am going to push to try to finish before Tuesday standup, but at the very worst Tuesday end of day.

**Comment by travislbrundage** [comment]
This is now mostly complete, just need to fill in with more images as desired!

**Comment by travislbrundage** [comment]
There is a small bug in the styling where the image seems to be scrolled down more than it should be. I am unsure of the cause currently. I estimate maximum one hour more to finish up.

**Comment by jonpmarino** [comment]
@reynalddizon should have imagery for you. We may need to generate a new set of GIFs to fit in this image strips throughout the parralax

**Comment by reynalddizon** [comment]
Yeah, Clarence and I talked about this last week and we've decided on adding a django data model for the parallax GIFs so we can just dump in new GIFs every time i create new ones. I just have a minor concern on what this might look like, (I've never used a GIF as a parallax image), this might cause motion sickness to viewers. But we'll see.

**Comment by jonpmarino** [comment]
yeah, make them sick!

On Tue, Jun 16, 2015 at 9:28 AM, reynalddizon <notifications@github.com>
wrote:

> Yeah, Clarence and I talked about this last week and we've decided on
> adding a django data model for the parallax GIFs so we can just dump in new
> GIFs every time i create new ones. I just have a minor concern on what this
> might look like, (I've never used a GIF as a parallax image), this might
> cause motion sickness to viewers. But we'll see.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/reynalddizon/MapStoryToDo/issues/17#issuecomment-112427668>
> .
>



-- 
Jonathan Parnell Marino
(815) 222 9088

**Comment by travislbrundage** [comment]
I think as long as the GIFs aren't too flashy, it should not really be a problem. :)

**Comment by travislbrundage** [comment]
This is basically finished. It'll be pushed tonight and I'll make one more comment with instructions on how to add more images/parallax sections. It should be very easy to add more parallax sections from now on.

**Comment by jonpmarino** [comment]
Awesome!!!

**Comment by travislbrundage** [comment]
Adding images
===========
1) Open the django admin panel (at /admin/)
2) Navigate to 'Parallax Images' (under MapStory)
3) Click 'Add'
4) Give the image you want to add a name and upload the file (the name is important for later)

Using images for parallax
===================
Use the following code template:
```
<section class="parallax">
<div data-stellar-background-ratio="0.5" data-stellar-vertical-offset="[number]" style="background-image: url({{ images|by_name:'[name]' }});">
</div>
</section>
```
Where [number] is any number of your choosing. This number just slides the image up or down at default. So, it lets you choose what part of the image to focus on.
[name] is the name you created for the image when uploading it into the django admin panel.

That should do it. You should be able to add that snippet of HTML anywhere in [index.html](https://github.com/MapStory/mapstory-geonode/blob/master/mapstory/templates/index.html) - if you want parallax images on other pages, just let me know and I can make the necessary adjustments.

I can go ahead and add whatever images you like too, just tell me what to put where. :)

**Comment by travislbrundage** [comment]
Closed by [Pull Request #30](https://github.com/MapStory/mapstory-geonode/pull/30)

**Comment by travislbrundage** [comment]
Reopening - fixing the page to be the old homepage design.
